test/unit/transactionRunner-test.js

```diff
@@ -279,11 +275,7 @@ describe('TransactionRunner', () => {
         beforeEach(() => {
           transaction.response.status = status;
           transaction.origin.filename = filename;
-
-          if (!runner.configuration.data) { runner.configuration.data = {}; }
-          if (!runner.configuration.data[filename]) { runner.configuration.data[filename] = {}; }
-          runner.configuration.data[filename].mediaType = 'application/swagger+json';
-
+          transaction.apiDescriptionMediaType = 'application/swagger+json';
```

Comment:
```
const filename = 'api-description.yml';
```

Shouldn't the media type be the YAML variant?

Reply:
For consistency, yes. Or the filename to be `.json`.

Reply:
Oh, looking into the code now and it's because Dredd takes whatever Fury returns as the first media type from the `.detect()` function. That way it always gets `application/swagger+json`, even for YAML files.

It's not an issue for Dredd, but it _is_ an issue once it's propagated to the transaction object. This is actually a blocking problem 😞

Reply:
Solved in a dirty way in 32f3864